### PR TITLE
Feature/fetch article data

### PR DESCRIPTION
**Pull request by wolf-jozef**
- Context for fetching data created,
- knowledge articles fetched and displayed in slider,

### CODE REVIEW

File: src/components/articleSection/components/ArticleSection.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    if (isAuthenticated) {
+      getAll().then((articles: KnowledgeArticleResponse) => {
+        setArticles(articles);
+      });
+    }
+  }, [isAuthenticated]);
+
+  const mostViewedArticles =
+    articles &&
+    articles.value.sort((a: any, b: any) => {
```
Comment (magnowak) [inline src/components/articleSection/components/ArticleSection.tsx:25]: let's use the web api inbuilt sorting option

File: src/components/articleSection/components/ArticleSlider.tsx [hunk 2]
```diff
@@ -54,7 +54,7 @@ const ArticleSlider = ({ children }: Props) => {
     autoplay: true,
     responsive: [
       {
-        breakpoint: 1024,
+        breakpoint: 1400,
```
Comment (magnowak) [inline src/components/articleSection/components/ArticleSlider.tsx:57]: Let's use standardized breakpoints, for example those suggested by MS: https://learn.microsoft.com/en-us/windows/apps/design/layout/screen-sizes-and-breakpoints-for-responsive-design

Let's define the breakpoints globally (eg. as css variables or anything that tailwind supports)  and use them in various components

File: src/components/PageLayout.tsx [hunk 3]
```diff
@@ -13,27 +13,27 @@ function PageLayout() {
...
-            redirect: "follow",
-          }
-        )
-          .then((response) => response.json())
-          .then((result) => setData(result.value))
-          .catch((error) => console.log("error", error));
-      });
-    }
-  }, [isAuthenticated]);
+  // useEffect(() => {
```
[the remark this answers is not in the document]
Comment (wolf-jozef) [reply src/components/PageLayout.tsx:16]: I am gonna delete whole file while moving the log in button logic to the navbar

**Review by sacktomasz — APPROVED** [review]

File: src/components/articleSection/components/ArticleSection.tsx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+import { useIsAuthenticated } from "@azure/msal-react";
+
+const ArticleSection = () => {
+  const { getAll } = useArticleService();
+  const [articles, setArticles] = useState<KnowledgeArticleResponse>();
+  const isAuthenticated = useIsAuthenticated();
+
+  useEffect(() => {
+    if (isAuthenticated) {
+      getAll().then((articles: KnowledgeArticleResponse) => {
```
[the remark this answers is not in the document]
Comment (wolf-jozef) [reply src/components/articleSection/components/ArticleSection.tsx:17]: Error catching display added

File: src/components/articleSection/components/ArticleSlider.tsx (hunk 2 above)
Comment (wolf-jozef) [reply src/components/articleSection/components/ArticleSlider.tsx:57]: ok

File: src/components/articleSection/components/ArticleSection.tsx (hunk 1 above)
Comment (wolf-jozef) [reply src/components/articleSection/components/ArticleSection.tsx:25]: ok

**Pull request merged**